### PR TITLE
Mobile

### PR DESCRIPTION
**Pull request by claudioccm**
Just a media query stylesheet, to make the interface more mobile friendly

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -6,6 +6,7 @@
 <!--[if gt IE 9]><!--> <html class="no-js" xml:lang="<%= locale %>" lang="<%= locale %>"> <!--<![endif]-->
 <head>
   <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
```
Comment (jpmckinney) [inline app/views/layouts/application.html.erb]: There's already a viewport meta tag lower down. You should delete that other one.

**Comment by jpmckinney** [comment]
One thing to fix. Otherwise looks good.

**Comment by lharv076** [comment]
@claudioccm Hey, what's the status with this, has the one thing been fixed? Can we deploy it this week?

**Comment by claudioccm** [comment]
yes. I fixed it before send the PR

**Comment by lharv076** [comment]
@claudioccm Ok great! @jpmckinney When can we deploy it?

**Comment by jpmckinney** [comment]
I believe it's now deployed. Let me know if there are any issues.

**Comment by lharv076** [comment]
Great! I'll do some tests this week. Thanks!

On Wed, Feb 17, 2016 at 12:47 PM, James McKinney <notifications@github.com>
wrote:

> I believe it's now deployed. Let me know if there are any issues.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opennorth/citizenbudgetapp.com/pull/77#issuecomment-185322448>
> .
>



-- 
*Laurence Harvey Leclerc*
Product Development Manager
1.888.750.4980 x. 706

Citizen Budget <http://www.citizenbudget.com> / Budget Citoyen
<http://www.budgetcitoyen.com>
opennorth.ca / nordouvert.ca

**Comment by lharv076** [comment]
Refer to test.citizenbudget.com to see the following issues live.


*In the desktop version:*

1. The images don't float automatically to the right anymore, and are
scaled at 60% width. Not a big deal for now, I just added a line of CSS to
our upcoming clients consultation.

2. The submit button and the dropdown widgets are really large now and the
submit button is not centered anymore.

3. The dropdown widget in the contact information section is massive.


*In the mobile version:*

1. When you initially access the consultation, its not automatically fitted
to the screen, I have to adjust it.

2. The balance bar isn't thick enough height-wise, the 0$ overlaps with the
black message bar.

3. Some of the questions are all pushed to the left and I can't see why.

4. Check box/boxes aren't showing up well, they're too small and not
aligned properly.

5. The radio-button size on the other hand is great, but they overlap with
the text.

6. All of the slider/on-off widgets are too small. I like the position
format, but the sliding bar should be bigger and take up the whole width of
the screen for easier manipulation.

7. The learn more pop-up is really thin width-wise. It would also be
essential that it disappears by clicking anywhere on the screen because it
hid the original learn more button, so now there's no way to make it
disappear.

8. All the images are placed awkwardly. I would place them right after or
before a section description, rather than in the  description box.

9. In the Contact information section, the text area widget should take up
the width of the screen (like the text box widget).

10. In the Contact information section, I like how the dropdown widget
options pop up when you click on it. I would make the initial box (before
clicking on it) take up the width of the screen as well though.


Let me know if some of these aren't clear!

Thanks,
Laurence

On Wed, Feb 17, 2016 at 12:58 PM, Laurence Harvey Leclerc <
laurence@opennorth.ca> wrote:

> Great! I'll do some tests this week. Thanks!
>
> On Wed, Feb 17, 2016 at 12:47 PM, James McKinney <notifications@github.com
> > wrote:
>
>> I believe it's now deployed. Let me know if there are any issues.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/opennorth/citizenbudgetapp.com/pull/77#issuecomment-185322448>
>> .
>>
>
>
>
> --
> *Laurence Harvey Leclerc*
> Product Development Manager
> 1.888.750.4980 x. 706
>
> Citizen Budget <http://www.citizenbudget.com> / Budget Citoyen
> <http://www.budgetcitoyen.com>
> opennorth.ca / nordouvert.ca
>



-- 
*Laurence Harvey Leclerc*
Product Development Manager
1.888.750.4980 x. 706

Citizen Budget <http://www.citizenbudget.com> / Budget Citoyen
<http://www.budgetcitoyen.com>
opennorth.ca / nordouvert.ca

**Comment by lharv076** [comment]
11. The property assessment value box is also pretty massive now. (example:
http://westkelowna.citizenbudget.com/?token=sSdbiwxm9YwqS173CbFP)

On Thu, Feb 18, 2016 at 10:27 AM, Laurence Harvey Leclerc <
laurence@opennorth.ca> wrote:

> Refer to test.citizenbudget.com to see the following issues live.
>
>
> *In the desktop version:*
>
> 1. The images don't float automatically to the right anymore, and are
> scaled at 60% width. Not a big deal for now, I just added a line of CSS to
> our upcoming clients consultation.
>
> 2. The submit button and the dropdown widgets are really large now and the
> submit button is not centered anymore.
>
> 3. The dropdown widget in the contact information section is massive.
>
>
> *In the mobile version:*
>
> 1. When you initially access the consultation, its not automatically
> fitted to the screen, I have to adjust it.
>
> 2. The balance bar isn't thick enough height-wise, the 0$ overlaps with
> the black message bar.
>
> 3. Some of the questions are all pushed to the left and I can't see why.
>
> 4. Check box/boxes aren't showing up well, they're too small and not
> aligned properly.
>
> 5. The radio-button size on the other hand is great, but they overlap with
> the text.
>
> 6. All of the slider/on-off widgets are too small. I like the position
> format, but the sliding bar should be bigger and take up the whole width of
> the screen for easier manipulation.
>
> 7. The learn more pop-up is really thin width-wise. It would also be
> essential that it disappears by clicking anywhere on the screen because it
> hid the original learn more button, so now there's no way to make it
> disappear.
>
> 8. All the images are placed awkwardly. I would place them right after or
> before a section description, rather than in the  description box.
>
> 9. In the Contact information section, the text area widget should take up
> the width of the screen (like the text box widget).
>
> 10. In the Contact information section, I like how the dropdown widget
> options pop up when you click on it. I would make the initial box (before
> clicking on it) take up the width of the screen as well though.
>
>
> Let me know if some of these aren't clear!
>
> Thanks,
> Laurence
>
> On Wed, Feb 17, 2016 at 12:58 PM, Laurence Harvey Leclerc <
> laurence@opennorth.ca> wrote:
>
>> Great! I'll do some tests this week. Thanks!
>>
>> On Wed, Feb 17, 2016 at 12:47 PM, James McKinney <
>> notifications@github.com> wrote:
>>
>>> I believe it's now deployed. Let me know if there are any issues.
>>>
>>> —
>>> Reply to this email directly or view it on GitHub
>>> <https://github.com/opennorth/citizenbudgetapp.com/pull/77#issuecomment-185322448>
>>> .
>>>
>>
>>
>>
>> --
>> *Laurence Harvey Leclerc*
>> Product Development Manager
>> 1.888.750.4980 x. 706
>>
>> Citizen Budget <http://www.citizenbudget.com> / Budget Citoyen
>> <http://www.budgetcitoyen.com>
>> opennorth.ca / nordouvert.ca
>>
>
>
>
> --
> *Laurence Harvey Leclerc*
> Product Development Manager
> 1.888.750.4980 x. 706
>
> Citizen Budget <http://www.citizenbudget.com> / Budget Citoyen
> <http://www.budgetcitoyen.com>
> opennorth.ca / nordouvert.ca
>



-- 
*Laurence Harvey Leclerc*
Product Development Manager
1.888.750.4980 x. 706

Citizen Budget <http://www.citizenbudget.com> / Budget Citoyen
<http://www.budgetcitoyen.com>
opennorth.ca / nordouvert.ca

**Comment by jpmckinney** [comment]
Should I revert the change for now?

**Comment by lharv076** [comment]
Yes please! That would be great.

Thanks,
Laurence

On Thu, Feb 18, 2016 at 11:35 AM, James McKinney <notifications@github.com>
wrote:

> Should I revert the change for now?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opennorth/citizenbudgetapp.com/pull/77#issuecomment-185805249>
> .
>



-- 
*Laurence Harvey Leclerc*
Product Development Manager
1.888.750.4980 x. 706

Citizen Budget <http://www.citizenbudget.com> / Budget Citoyen
<http://www.budgetcitoyen.com>
opennorth.ca / nordouvert.ca

**Comment by lharv076** [comment]
@claudioccm Hey Claudio,

Can we fix these issues by the end of next week?

Thanks!
Laurence

**Comment by jpmckinney** [comment]
Done

**Pull request merged**